spring-cloud-task-core/pom.xml

```diff
@@ -58,6 +66,17 @@
 			<artifactId>mockito-core</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.hsqldb</groupId>
+			<artifactId>hsqldb</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-dbcp2</artifactId>
+			<scope>test</scope>
+		</dependency>
```

Comment:
This is being nit picky, but since Boot prefers the Tomcat connection pool, we probably should use it as well (in fact, they list DBCP2 as the last option they'd use in their docs here: https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-sql.html).